src/components/Calculator.js

```diff
@@ -1,70 +1,178 @@
 import React, { Component } from 'react';
+import calculate from '../logic/calculate';
 
 // eslint-disable-next-line react/prefer-stateless-function
 class Calculator extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {};
+    this.myHandleClick = this.myHandleClick.bind(this);
+  }
+
+  myHandleClick(event) {
+    this.setState((state) => calculate(state, event.target.textContent));
+  }
```

Comment:
- [ ] While trying to use your calculator, I noticed that after performing some actions on the calculator(e.g after adding two digits), I can't seem to perform any other operation at all as the console shows an error saying "unknown operation".  Kindly check it out 👍🏽 


https://user-images.githubusercontent.com/63473584/188997488-e03c44af-6e23-47d3-997e-f5c290829577.mp4

![image](https://user-images.githubusercontent.com/63473584/188997172-60457bb7-bd69-435e-a722-59aca9b48d53.png)